### PR TITLE
v0.11.1 Release

### PR DESCRIPTION
**Pull request by cjstehno**
* Support for OkHttp client library
* Build enhancements
* Test refactoring
* Added and modified documentation
* Added more unit tests

### CODE REVIEW

**Comment by cjstehno** [comment]
Odd, not sure where the conflicts are coming from since this code was branched off of master, which has not changed... looks like I have some git surgery to do.

**Comment by cjstehno** [comment]
Ok, I think we are ready to go... I will let it bake a bit in case you want to review/test it. When its ready I will merge and do the publishing/deployments, etc.

File: http-builder-ng-apache/src/test/groovy/groovyx/net/http/UsesApacheClient.groovy [hunk 1]
```diff
@@ -18,25 +18,11 @@ package groovyx.net.http
 import java.util.function.Function
 
 /**
- * Helpers for testing with the Apache client HttpBuilder.
+ * Trait used to denote testing with the Apache client library.
```
Comment (dwclark) [inline http-builder-ng-apache/src/test/groovy/groovyx/net/http/UsesApacheClient.groovy]: @cjstehno  Woohoo! A traits success story!

**Comment by dwclark** [comment]
@cjstehno Looks good.

It looked like OkHttp support was pretty straightforward to integrate.

Great work on making the tests re-usable. Being able to run the exact same tests for all clients is a huge win. Merge it whenever you are ready.

**Comment by cjstehno** [comment]
Awesome. I will merge and publish the release over the weekend (after I resolve the conflicts I created patching the README :-) )

**Pull request merged**